README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
It's good that it always works for local js/css files. But in the future, other developers may add new non-local js/css dependency. So it should have a document to guide developers on what to do if they introduce new non-local js/css depedency (e.g. if everything is static, then can download the dependency as a local file and put into what folder.)